pkg/controller/master/setting/harvester_rancher_sync.go

```diff
@@ -0,0 +1,58 @@
+package setting
+
+import (
+	"reflect"
+
+	ranchersettings "github.com/rancher/rancher/pkg/settings"
+
+	harvesterv1 "github.com/harvester/harvester/pkg/apis/harvesterhci.io/v1beta1"
+	"github.com/harvester/harvester/pkg/settings"
+)
+
+func (h *Handler) syncHarvesterToRancher(setting *harvesterv1.Setting) error {
+	switch setting.Name {
+	case settings.UIIndexSettingName:
+		return h.copyHarvesterToRancher(setting, ranchersettings.UIIndex.Name)
```

Comment:
based on the PR description the harvester `ui-index` setting should be mapped to rancher `ui-dashboard-index` setting.

however we are here referring the rancher `ui-index` setting. @WuJun2016 would you please be able to confirm what the correct setting needs to be?

Reply:
based on the PR description the harvester `ui-index` setting should be mapped to rancher `ui-dashboard-index` setting.

however we are here referring the rancher `ui-index` setting. @WuJun2016 would you please be able to confirm what the correct setting needs to be?

Reply:
should be `ui-dashboard-index`

Reply:
> Case 1: Harvester ui-index maps to Rancher ui-dashboard-index
>   1. Change Harvester ui-index. Rancher ui-dashboard-index should also be changed.
>   2. Set default to Rancher ui-dashboard-index. Harvester ui-index should also be set to the default value.

I think what is described in this test step is consistent with the requirements of the UI.

Reply:
fixed